CHANGELOG.md

```diff
@@ -1,6 +1,23 @@
 Changelog
 =========
 
+## 6.1.8 (2020-10-07)
+
+### Enhancements
+
+* An invalid or missing API key now causes an NSException to be thrown upon
+  calling [BugsnagClient start].
```

Comment:
```suggestion
  calling [Bugsnag start].
```

Reply:
Also, I think we should probably say a bit more about the change so it's easier for people to find details – i.e. the check is moved from creation the configuration object.